device/alpaka/include/traccc/alpaka/utils/get_vecmem_resource.hpp

```diff
@@ -58,49 +58,49 @@ class copy;
 }  // namespace sycl
 }  // namespace vecmem
 
-namespace traccc::alpaka::vecmem {
+namespace traccc::alpaka::vecmem_resources {
```

Comment:
As long as we're changing things anyway: Do we need an extra namespace here at all? I myself would be fairly happy with just the `traccc::alpaka::host_device_types<...>`  type names. 🤔

I could possibly imagine a `traccc::alpaka::details::` namespace, in case clients of the library don't need to interact with these types normally. But other sub-namespaces should probably not be invented. 🤔

Reply:
Possibly!

I just wasn't a fan of the `traccc::alpaka::vecmem` namespace clobbering the `vecmem::` namespace in certain cases, so I changed the name. I'll test some bits out and see what feels more natural.